tests/flow/test_function_calls.py

```diff
@@ -168,11 +168,21 @@ def test09_static_aggregation(self):
         expected_result = [[2]]
         self.env.assertEquals(actual_result.result_set, expected_result)
 
+        query = "RETURN max([2])"
+        actual_result = graph.query(query)
+        expected_result = [[[2]]]
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
         query = "RETURN min(3)"
         actual_result = graph.query(query)
         expected_result = [[3]]
         self.env.assertEquals(actual_result.result_set, expected_result)
 
+        query = "RETURN min([3])"
+        actual_result = graph.query(query)
+        expected_result = [[[3]]]
+        self.env.assertEquals(actual_result.result_set, expected_result)
```

Comment:
Please add a test which process multiple arrays in addition to a test that process a mixture of data types:
e.g.
```
UNWIND [[1], [2], [2], [1]] AS x RETURN max(x), min(x)
UNWIND [1, 2, '1' ,'2' ,[1] ,[2] ,1 ,2, '1', '2', NULL, True] AS x RETURN max(x), min(x)
```